data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_calculations.cfg

```diff
@@ -95,9 +95,9 @@
 # API(s) being tested: [attacks]sub=
 ##
 # Expected end state:
-# Bob's attack has 7 strikes. The difference to add2_add3_cumulative is that sub chooses the lower value, and the order of the abilities determines the order of the calculations.
+# Bob's attack has 10 strikes. The difference to add2_add3_cumulative is that add and sub are both used.
```

Comment:
Technically, that's not a difference.

Bob's attack has 10 strikes. Both the add and the negative sub are used.

Reply:
Fixed